packages/shared/src/components/plus/PlusComparingCards.tsx

```diff
@@ -141,22 +148,22 @@ const PlusCard = ({
           variant={ButtonVariant.Primary}
           data-funnel-track={FunnelTargetId.StepCta}
         >
-          Get started
+          {copy?.cta || 'Get started'}
         </Button>
       )}
       <Typography
         className="mb-4 text-center"
         color={TypographyColor.Tertiary}
         type={TypographyType.Footnote}
       >
-        {isPaidPlan ? (
-          <>
-            30 day hassle-free refund.{' '}
-            <span className="whitespace-nowrap">No questions asked.</span>
-          </>
-        ) : (
-          'Free forever'
-        )}
+        {isPaidPlan
+          ? copy?.description || (
+              <>
+                30 day hassle-free refund.{' '}
+                <span className="whitespace-nowrap">No questions asked.</span>
+              </>
+            )
+          : copy?.description || 'Free forever'}
```

Comment:
:lolsob:

Reply:
Pretty much a pain..
but until we move this fully to custom Freyja this was the easiest 😅